vital/video_metadata/pos_metadata_io.cxx

```diff
@@ -111,7 +112,8 @@ read_pos_file( path_t const& file_path )
   md->add( NEW_METADATA_ITEM( VITAL_META_SENSOR_PITCH_ANGLE, std::stod( tokens[ base + 1] ) ) );
   md->add( NEW_METADATA_ITEM( VITAL_META_SENSOR_ROLL_ANGLE, std::stod( tokens[base + 2] ) ) );
 
-  kwiver::vital::geo_lat_lon latlon( std::stod( tokens[ base + 3]), std::stod( tokens[ base + 4 ] ) );
+  kwiver::vital::vector_2d raw_latlon{ std::stod( tokens[ base + 4 ]), std::stod( tokens[ base + 3 ] ) };
+  kwiver::vital::geo_point latlon{ raw_latlon, SRID::lat_lon_WGS84 };
```

Comment:
>  Should it be `lonlat`?

I suppose that's plausible.

> and `SRID::lon_lat_WGS84`?

That I'm less sure. This is an API name (and very possibly should be changed in a separate PR, if we do change it), and is intended to refer to the projection system (e.g. UTM, UPS, lat/lon, ECEF). English speakers usually think of the projection system as "latitude and longitude", not "longitude and latitude". OTOH, folks can probably figure it out...

Incidentally, PROJ.4 also processes lat/lon in easting, northing, height order while still using "latlong" in API's.